docs/_includes/calendar.html

```diff
@@ -0,0 +1,396 @@
+            </table>
+        </div>
+    </div>
+</div>
+    {% endhighlight %}
+
+    <h3>Calendar with Disabled Cells and "Today"</h3>
+
+    <p>Each cell can be disabled, and the current day can be marked with aria roles.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="calendar">
+                <div class="calendar__body">
+                    <div class="calendar__month">
+                        {% include calendar-month.html index="1" %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="calendar">
+    <div class="calendar__body">
+        <div class="calendar__month">
+            <table>
+                <caption>
+                    August 2019
+                </caption>
+                <thead>
+                    <tr>
+                        <th>Su</th><th>Mo</th><th>Tu</th><th>We</th><th>Th</th><th>Fr</th><th>Sa</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td colspan="4"></td>
+                        <td><span class="calendar__cell--disabled">1</span></td>
+                        <td><span class="calendar__cell--disabled">2</span></td>
+                        <td><span class="calendar__cell--disabled">3</span></td>
+                    </tr>
+                    ...
+                    <tr>
+                        <td><span class="calendar__cell--disabled">11</span></td>
+                        <td><span class="calendar__cell--disabled">12</span></td>
+                        <td><span class="calendar__cell--disabled">13</span></td>
+                        <td aria-current="date">
+                            <button>14
+                        </button></td>
+                        <td><span>15</span></td>
+                        <td><span>16</span></td>
+                        <td><span>17</span></td>
+                    </tr>
+                    ...
+                    <tr>
+                        <td><span>25</span></td>
+                        <td><span>26</span></td>
+                        <td><span>27</span></td>
+                        <td><span>28</span></td>
+                        <td><span>29</span></td>
+                        <td><span>30</span></td>
+                        <td><span>31</span></td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</div>
+    {% endhighlight %}
+
+    <h3>Interactive Calendar</h3>
+
+    <p>To make a calendar interactive, replace the <code>span</code> elements with <code>button</code>. Buttons are disabled directly instead of with a class, but all other attributes remain the same between static and interactive calendars. In interactive calendars, one and only one button should have a <code>tabindex</code> in the table. The rest need to set <code>tabindex</code> to <code>-1</code>.</p>
```

Comment:
To keep things simpler, I would probably leave off all the wording and examples regarding tabindex and just say that keyboard focus management should be handled via javascript and a roving tab index.